packages/mdc-textfield/_mixins.scss

```diff
@@ -176,12 +178,16 @@
 }
 
 @mixin mdc-text-field-dense_ {
-  // NOTE: This is an eyeball'd approximation of what's in the mocks.
-  @include mdc-floating-label-float-position(110%, 0%, $mdc-text-field-dense-label-scale);
+  @include mdc-floating-label-float-position($mdc-text-field-dense-label-position-y, 0%, $mdc-text-field-dense-label-scale);
```

Comment:
This was changed from 110% to 70%?

Reply:
110% was the value for the variant being deprecated and the new default has a value of 70%. The only two variants this affects is the outlined and the "box" however the outlined has its own value in dense.